### PR TITLE
Compat doc: GetECDsaPublicKey correctly handles brainpool curves

### PR DESCRIPTION
**Pull request by krwq**

### CODE REVIEW

File: Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Minor
+
+### Version Introduced
+4.8
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Starting with the .NET Framework 4.8, the `System.Security.Cryptography.X509Certificates.ECDsaCertificateExtensions.GetECDsaPublicKey(this System.Security.Cryptography.X509Certificates.X509Certificate2 certificate)` will now correctly handle Brainpool elliptic curves.
```
Comment (bartonjs) [inline Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md]: While "Brainpool" is fine for the title, really the change was to handle all named curves that Windows 10 supports (that Windows 8.1 did not).

So perhaps "will now correctly handle all named curves supported by the OS, including the Brainpool elliptic curves."

Comment (rpetrusha) [inline Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md]: Is it useful to add a one-sentence description of the previous behavior? ("In .NET Framework 4.7.2 and earlier versions...")

File: Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+### Recommended Action
+If you encounter compatibility issues with this change on the .NET Framework 4.8 or a later version, you can opt-out of it by adding the following line to the <runtime> section of the app.config file:
```
Comment (rpetrusha) [inline Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md]:
```suggestion
If you encounter compatibility issues with this change on the .NET Framework 4.8 or a later version, you can opt-out of it by adding the following line to the \<runtime> section of the app.config file:
```

File: Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+```xml
+<appSettings>
+    <add key="Switch.System.Security.Cryptography.X509Certificates.ECDsaCertificateExtensions.UseLegacyPublicKeyReader" value="true" />
+</appSettings>
+```
+
+### Category
```
Comment (rpetrusha) [inline Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md]: Before category, you also need to add Affected APIs:

> \#\#\# Affected APIs
> \* \`M:System.Security.Cryptography.X509Certificates.ECDsaCertificateExtensions.GetECDsaPublicKey(System.Security.Cryptography.X509Certificates.X509Certificate2)\`
>

File: Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md (hunk 1 above)
Comment (krwq) [reply Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md]: previously it just didn't work with this API :smile:

File: Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md (hunk 2 above)
Comment (krwq) [reply Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md]: why is this needed? should this be applied to both `<` and `>`?

Comment (rpetrusha) [reply Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md]: '<` needs to be escaped, but `>` doesn't.

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @krwq. I'll merge your PR now.

**Pull request merged**